Src/base/DisplayManager.cpp

```diff
@@ -2616,7 +2616,6 @@ void DisplayManager::markBootFinished(bool finished)
 				|| currentState() == DisplayStateOnPuck
 				|| currentState() == DisplayStateDockMode
 				|| (currentState() == DisplayStateDim && !Settings::LunaSettings()->turnOffAccelWhenDimmed))
-		orientationSensorOn();
```

Comment:
no, you'll have to remove the if too, otherwise the if will apply to the next instruction (requesCurrentLocation)

Reply:
@tofee I shouldn't do this without enough coffee. Coding style here leaves to be desired too ;) Anyway removed now.